src/utils/window-console.js

```diff
@@ -46,7 +46,12 @@ export function addDataToWindowObject(
   defineProperty(target, 'filteredMarkers', {
     enumerable: true,
     get() {
-      return selectors.selectedThread.getPreviewFilteredMarkers(getState());
+      const state = getState();
```

Comment:
Optional: I feel like this should be a selector instead `getPreviewFilteredMarkersForConsole`.

Reply:
I've mixed feelings about that. So I tried with my profile with a lot of markers, and I've seen that this takes ~20ms on my machine, which seems fast enough for this pathological case. So I don't think we need memoization.
Then I'd like to avoid selectors returning markers directly, unless we see it's used in several places (like getSelelectedMarker). So for now I won't implement a selector for this specific case, but I'd be happy to revisit with a convincing case.

Reply:
Sounds good to me. Thanks for checking the speed.